Framework/Algorithms/src/DetectorEfficiencyCor.cpp

```diff
@@ -245,9 +251,9 @@ void DetectorEfficiencyCor::correctForEfficiency(
     // now get the sin of the angle, it's the magnitude of the cross product of
```

Comment:
You can remove all the code computing `theta` and just use `0.5 * detectorInfo.twoTheta(detIndex)`.